I fixed a typo in EnvSearchServiceInternalPort that was EnvSrchServiceInternalPort

AmineAouragh opened the pull request.

BastLeblanc commented:
Hi, thanks for your pull request, before accepting it, can you check these:

- if it's solving an issue can you reference it and describe it. Or explain that it's just cosmetic on a typo.
- have you tried deploying the project with your changes? It seems that you haven't modified the name in all the right places.
- ensure the integration tests are all passing (see https://github.com/aws-samples/retail-demo-store/tree/master/src/run-tests for how to run the integration tests)

Looking forward for your answers

AmineAouragh commented:
Hey Bastien! Thanks for the reply. I will make sure of that and fix everything I missed regarding that issue. I think I'll need to add more details in the pull request description. Thank you.

AmineAouragh commented:
I just fixed the same typo - 'Serch' should be 'Search in 3 different locations in the _template.yaml file here -> [retail-demo-store/aws/cloudformation-templates/services/service/_template.yaml](url)

Here: 
![2023-10-18 13_34_37-I fixed a typo in EnvSearchServiceInternalPort that was EnvSrchServiceInternalPo](https://github.com/aws-samples/retail-demo-store/assets/71039249/563a1cb4-bfdf-4571-87d3-e58b34b6aee1)

and here:
![2023-10-18 13_38_05-I fixed a typo in EnvSearchServiceInternalPort that was EnvSrchServiceInternalPo](https://github.com/aws-samples/retail-demo-store/assets/71039249/dcdf24f8-b421-4713-a051-2a122648a9c0)

AmineAouragh commented:
More details were added to the pull request description

BastLeblanc commented:
@AmineAouragh the deployment fails after your proposed PR with the error for each service:
`Parameters: [EnvSearchServiceInternalPort] do not exist in the template`

Please do a full end to end testing of deploying the cloudformation template before proposing a PR. thanks in advance.

BastLeblanc commented:
I'm closing this PR, please reopen if you get to run the tests before proposing the PR. Thanks
